docs/maps/asset-tracking-tutorial.asciidoc

```diff
@@ -270,47 +270,39 @@ PUT _ingest/pipeline/tri_met_tracks
 ----------------------------------
 
 [float]
-==== Step 2: Start {filebeat}
+==== Step 2: Configure {agent}
 
-. Replace the contents in your `filebeat.yml` file with the following:
-+
-[source,yaml]
-----------------------------------
-filebeat.inputs:
-# Fetch trimet bus data every minute.
-- type: httpjson
-  interval: 1m
-  request.url: "https://developer.trimet.org/ws/v2/vehicles?appID=<tri_met_app_id>"
-  response.split:
-    target: body.resultSet.vehicle
-  processors:
-    - decode_json_fields:
-        fields: ["message"]
-        target: "trimet"
+. From the {kib} main menu, click *Fleet*, then the *Agent policies* tab.
 
-  pipeline: "tri_met_tracks"
+. Click the name of the agent policy where you want to add the *Custom API* integration. The configuration changes you make only apply to the policy you select.
 
+. Click the name of the *Custom API* integration, or add the integration if the agent policy does not yet have it.
 
-# ---------------------------- Elastic Cloud Output ----------------------------
-cloud.id: <cloud_id>
-cloud.auth: <username:password>
+. From the *Edit Custom API integration* page, expand the *Change defaults* section.
 
-----------------------------------
+. Set the *Dataset name* to *httpjson.trimet*.
+
+. Set the *Ingest Pipeline* to *tri_met_pipeline*.
 
-. Replace `<tri_met_app_id>` with your TriMet application id.
-. Replace `<username:password>` with your Elastic Cloud deployment credentials.
-. Replace `<cloud_id>` with your {ece}/ece-cloud-id.html[elastic cloud id].
-. Open a terminal window, and then navigate to the {filebeat} folder.
-. In your `filebeat` folder, run {filebeat} with the edited config:
+. Set the *Request URL* to *https://developer.trimet.org/ws/v2/vehicles?appID=<tri_met_app_id>*.
+
+. Set *Response Split* to *target: body.resultSet.vehicle*.
+
+. At the bottom of the configuration, expand *Advanced options*.
+
+. Set *Processors* to:
 +
-[source,bash]
+[source,yaml]
 ----------------------------------
-/bin/filebeat -c filebeat.yml
+- decode_json_fields:
+    fields: ["message"]
+    target: "trimet"
 ----------------------------------
 
-. Wait for {filebeat} to start shipping data to Elastic Cloud. {filebeat} should not produce any output to stdout. 
+. Leave everything else as defaults.
+
+. Click *Save integration* to deploy the configuration to any {agent} with the policy assigned.
```

Comment:
How will this work if multiple agents try to ingest from REST API?

Reply:
We are going to have data duplication.